scripts/unpack_ddf.py

```diff
@@ -177,7 +177,7 @@ def print_shader(shader):
     print("{")
     for field, data in shader.ListFields():
         if field.name == 'source':
-            lines = str(data.strip(), encoding="ascii").split("\n")
```

Comment:
it's not really related fix, but it is small tool fix we need to be able to unpack SPIRV shaders